packages/babel-generator/src/printer.ts

```diff
@@ -108,12 +107,13 @@ export type PrintJoinOptions = PrintListOptions & PrintSequenceOptions;
 class Printer {
   constructor(format: Format, map: SourceMap) {
     this.format = format;
-    this._buf = new Buffer(map);
 
     this._indentChar = format.indent.style.charCodeAt(0);
     this._indentRepeat = format.indent.style.length;
 
     this._inputMap = map?._inputMap;
+
+    this._buf = new Buffer(map, this._indentChar);
```

Comment:
Could we already pass `String.fromCharCode(this._indentChar)` here, so that then we don't have to convert it multiple times inside `Buffer`?